src/components/Ingredients.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+class Ingredients extends React.Component {
```

Comment:
Are these the missing ingredients? 
It's becoming confusing with the one on the list.

Reply:
Missing ingredients??

Reply:
This is the ingredients for the recipe

Reply:
The recipe contains ingredients and instructions. Separate API calls

Reply:
Ah ok. Cool
So I need to rename the single ingredient with 'currentIngredient' maybe..

Reply:
Ah I see. Sounds good.